apps/web/src/pages/brand/BrandPage.tsx

```diff
@@ -7,8 +7,10 @@ import PageContainer from '../../components/layout/components/PageContainer';
 import { Tabs } from '../../design-system';
 import { AuthContext } from '../../store/authContext';
 import { BrandingForm } from './tabs';
+import { LayoutsListPage } from './tabs/LayoutsListPage';
```

Comment:
Is there an index inside of `./tabs`? We could just have one import if so.

Reply:
Yes! There is. Forgot to add it